test/Api.Test/KeyManagement/Validators/SendRotationValidatorTests.cs

```diff
@@ -1,7 +1,7 @@
 ﻿using System.Text.Json;
+using Bit.Api.KeyManagement.Validators;
 using Bit.Api.Tools.Models;
 using Bit.Api.Tools.Models.Request;
-using Bit.Api.Tools.Validators;
```

Comment:
⚠️  Namespace within this file should also be updated

Reply:
Fixed this, and another instance of forgetting to change the namespace. Thank you!